### PR TITLE
Portfolio a

### PR DESCRIPTION
**Pull request by CSD-Ahmed**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+    font-weight: 700;
+    position: relative;
+}
+.section-title h2::before{
+    content: '';
+    height: 4px;
+    width: 50px;
+    background-color: var(--skin-color);
+    position: absolute;
+    top: 100%;
```
Comment (karwan01) [inline style.css:75]: check it again

File: style.css [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+    left: 0;
+}
+.section-title h2::after{
+    content: '';
+    height: 4px;
+    width: 25px;
+    background-color: var(--skin-color);
+    position: absolute;
+    top: 100%;
```
Comment (karwan01) [inline style.css:84]: test it with different value

File: index.html [hunk 3]
```diff
@@ -3,9 +3,71 @@
...
+                <p>I,m a web Designer with extensive experience for 3 years.my expert create and website design,Robotic creator and many  more...</p>
+                <a href="#contact" class="btn hire-me">Hire Me</a>
+            </div>
+            <div class="home-img padd-15">
+                <img src="./assets/pexels-photo-1212693.jpeg" alt="">
+            </div>
+        </div>
+    </div>
+</section>
+            <!--Home sectiono End-->
```
[the remark this answers is not in the document]
Comment (CSD-Ahmed) [reply index.html]: @karwan01 prettier added

**Review by karwan01 — APPROVED** [review]
Dast xosh kak @CSD-Ahmed

**Pull request merged**